include/CppUTest/CppUTestConfig.h

```diff
@@ -199,6 +199,27 @@
 #define CPPUTEST_CHAR_BIT 8
 #endif
 
+/* Support for "long long" type */
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
If CPPUTEST_USE_LONG_LONG is not #defined, then the header will contain a struct to the assert, and therefore you will still get a compiler error.... even when your code does support long long.